lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -1,9 +1,26 @@
 define((require, exports, module) ->
   Behaviour = require("ace/mode/behaviour").Behaviour
+  COMMENT_REGEX = /comment/
+  ESCAPE_REGEX = /escape/
+  MATH_REGEX = /math/
+  EQUATION_REGEX = /equation/
+  RPAREN_REGEX = /rparen/
+  correspondingClosing = {
+      '(': ')',
+      '[': ']'
+      '{': '}'
+  }
+
+
+  isCommentToken = (token, column) -> token? and COMMENT_REGEX.test(token.type) and column != 0
```

Comment:
Is is intentional?  `column != 0` 
If it is then maybe write a short comment and move this test to the second or even the first place, because it is cheaper than regexp test.